### DISCUSSION TITLE
LadeSteuerung: Änderung der festen Ladeleistung?

### DISCUSSION THREAD

**Discussion by wiggal** [wiggal/GEN24_Ladesteuerung#26] [Ideas]
Mit Einstellung der "Ladegrenze" im Reiter "LadeSteuerung" kann die Ladeleistung im Verhältnis zur definierten maximalen Ladeleistung aus der config.ini ("MaxLadung") fest eingestellt werden.

![grafik](https://github.com/wiggal/GEN24_Ladesteuerung/assets/69506553/eb1065e5-e20c-42b2-8645-cbdd31d39492)

Benutzt jemand diese Einstellungen? 

Wenn Ja, wäre es besser, wenn ich die einstellbare Ladeleistung auf die gesamt mögliche Ladeleistung des Wechselrichters beziehen würde? Evtl. mit einer prozentualen Abstufung ähnlich der Entladesteuerung, vielleicht mit 0,33%,66%,100%?

![grafik](https://github.com/wiggal/GEN24_Ladesteuerung/assets/69506553/bbf63a10-1baa-492d-a976-6b1bb04455fc)

Vielleicht bekomme ich hier Meinungen dazu.

**Comment by jaydee73** [comment]
Ich möchte diese Einstellung benutzen. ;-)

Grundsätzlich finde ich eine Anlehnung an die maximale Ladeleistung des WR etwas intuitiver als die Anlehnung an den Wert "MaxLadung". Aber sofern es entsprechend dokumentiert ist, auf was sich die Ladeleistung bezieht, könnte ich auch mit dem Bezug zu MaxLadung leben.

Eine prozentuale Abstufung, wie bei der Entladesteuerung, finde ich sinnvoll.

**Comment by mosjka1** [comment]
Ich benutze ebenfalls die Entladesteuerung. Sofern möglich, bitte die MaxLadung von der festen Entladegrenze abkoppeln. Die feste Entladegrenze 0, 20% 40%...100% soll sich auf die Entladelistung des Wechselrichters beziehen.

Evtl. noch vor die Prozente einen neuen Button "AUTO"(holt den Wert "MaxLadung" aus der config) einbauen. Oder ums einfacher zu machen, die "MaxLadung" aus der config entfernen und die Vorgabe dieser aus "Feste Entladegrenze %" heranziehen.

**Reply by wiggal** [reply]
Hi, 
Bei der **Entladesteuerung** beziehen sich die Prozentangeben bereits auf die Maximale Entladeleistung des Wechserichters, da in der config.ini keine Variabel für maximale Entladung existiert.

Die Frage ist hier, wie wir das bei der **Ladesteuerung** für die manuellen Ladegrenze handhaben sollen.

**Comment by wiggal** [comment]
> 
> Wenn Ja, wäre es besser, wenn ich die einstellbare Ladeleistung auf die gesamt mögliche Ladeleistung des Wechselrichters beziehen würde? Evtl. mit einer prozentualen Abstufung ähnlich der Entladesteuerung, vielleicht mit 0,33%,66%,100%?
> 

Ich habe mir die Möglichkeit mal genauer angesehen, und bin auf folgendes Problem gestoßen.

Die Ladeleistung meines WR liegt bei 10240Watt. In der Kombination mit meiem Akku kann aber nur mit 4500 Watt geladen werden.
Wenn ich nun 100% bzw. 50% der Ladeleistung des Wechselrichers einstelle ist dies immer noch begrenzt durch die 4500 Watt. Due wirkliche Ladeleistung (bei mir 4500W) kann ich aber nicht auslesen.
Das heißt je nachdem welche Batterie am WR hängt ist die Ladeleistung nicht kalkulierbar.

Deshalb tendiere ich dazu die Einstellung "HALB und VOLL" weiterhin auf die Variable  "MaxLadung" zu beziehen.

**Reply by jaydee73** [reply]
Verständlich. Den Gedankengang hatte ich auch schon. Es ist auch nicht nur von der Batterie abhängig, sondern auch vom Gen24-Modell selbst. In dieser pdf ist das zu sehen: https://www.photovoltaikforum.com/core/attachment/197915-se-sols-fronius-storage-solution-de-pdf/ (Seite 2).

Insofern: Ja, dann würde ich das auch so sehen, es weiterhin auf MaxLadung zu beziehen.
